artifactory/cli.go

```diff
@@ -605,6 +610,58 @@ func GetCommands() []cli.Command {
 				return pipInstallCmd(c)
 			},
 		},
+		{
+			Name:            "repo-template",
+			Aliases:         []string{"rpt"},
+			Usage:           repotemplate.Description,
+			HelpName:        common.CreateUsage("rt rpt", repotemplate.Description, repotemplate.Usage),
+			UsageText:       repotemplate.Arguments,
+			ArgsUsage:       common.CreateEnvVars(),
+			SkipFlagParsing: true,
+			BashComplete:    common.CreateBashCompletionFunc(),
+			Action: func(c *cli.Context) error {
+				return repoTemplateCmd(c)
+			},
+		},
+		{
+			Name:         "repo-create",
+			Aliases:      []string{"rc"},
+			Flags:        getTemplateUsersFlags(),
+			Usage:        repocreate.Description,
+			HelpName:     common.CreateUsage("rt rc", repocreate.Description, repocreate.Usage),
+			UsageText:    repocreate.Arguments,
+			ArgsUsage:    common.CreateEnvVars(),
+			BashComplete: common.CreateBashCompletionFunc(),
+			Action: func(c *cli.Context) error {
+				return repoCmd(c, false)
+			},
+		},
+		{
+			Name:         "repo-update",
+			Aliases:      []string{"ru"},
+			Flags:        getTemplateUsersFlags(),
+			Usage:        repoupdate.Description,
+			HelpName:     common.CreateUsage("rt ru", repoupdate.Description, repoupdate.Usage),
+			UsageText:    repoupdate.Arguments,
+			ArgsUsage:    common.CreateEnvVars(),
+			BashComplete: common.CreateBashCompletionFunc(),
+			Action: func(c *cli.Context) error {
+				return repoCmd(c, true)
+			},
+		},
+		{
+			Name:         "repo-delete",
+			Aliases:      []string{"rd"},
```

Comment:
rd --> rdel
(to be consistent with our other delete commands).
Let's change in help.go as well.